appinfo/info.xml

```diff
@@ -3,14 +3,13 @@
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>calendar</id>
 	<name>Calendar</name>
-	<summary>Calendar GUI for Nextcloud's CalDAV server</summary>
-	<description><![CDATA[A calendar app for Nextcloud. Easily sync events from various devices with your Nextcloud and edit them online.
+	<summary>A calendar app for Nextcloud.</summary>
```

Comment:
- App names are capitalized
- No need to mention »Nextcloud« in the summary of a Nextcloud app ;)